browser.go

```diff
@@ -27,29 +28,42 @@ func Terminate() error {
 	return nil
 }
 
+func resolveCanvas() *dom.HTMLCanvasElement, bool {
+	canvas, ok := document.QuerySelector("#glfw").(*dom.HTMLCanvasElement)
+	if ok == false {
```

Comment:
nit (not important to fix here though): You can just do `if !ok {`

Reply:
I was feeling a bit more verbose I guess.

Reply:
:)